src/invariant/test/AccountSubEntriesCountIsValidTests.cpp

```diff
@@ -158,6 +164,12 @@ addRandomSubEntryToAccount(Application& app, LedgerEntry& le,
     if (addSigner)
     {
         acc.signers.push_back(validSignerGenerator());
+        if (acc.ext.v() == 1 && acc.ext.v1().ext.v() == 2)
+        {
+            acc.ext.v1().ext.v2().signerSponsoringIDs.push_back(
+                autocheck::generator<SponsorshipDescriptor>()(5));
```

Comment:
Just curious, why do we pass in 5 to the generator? I noticed we do this in other generator calls as well.

Reply:
It's a good question, one I actually found myself asking yesterday when I wrote this although I didn't investigate at the time. Your question prompted me to take a real look. The relevant code is `generator_t` in xdrpp/autocheck.h. Some lines, such as 120, make me think that 5 might not be a good choice. Can you open an issue for future investigation?

Either way, I don't think this should stop us from merging this PR.

Reply:
Opened an issue - https://github.com/stellar/stellar-core/issues/2680. I agree that this doesn't affect the PR.